src/Atlas.cc

```diff
@@ -352,6 +354,17 @@ void Atlas::PostLoad()
     mvpBackupMaps.clear();
 }
 
+void Atlas::SaveAtlas(string pathSaveFileName, string strVocabularyName, string strVocabularyChecksum) {
+    unique_lock<mutex> lock(mMutexAtlas);
```

Comment:
[opt] `lock_guard` is preferred over `unique_lock` when you're not going to call `unlock()`.